README.md

```diff
@@ -0,0 +1,51 @@
+Automation wrapper for burst_io
+
+Description:
+
+        This wrapper runs a program that performs burst io activity.
+        The program will perform a specified IO for a designated amount
+        of time and then sleeps for another amount of time.  Once the program
+        wakes up, it will terminate if the overall time of the run has expired,
+        else it will perform another IO iteration.
+        and does not deal with numa.
+  
+Location of underlying workload: part of the github kit
+
+Packages required: gcc,bc,wget
+
+```
+To run:
+[root@hawkeye ~]# git clone https://github.com/redhat-performance/io_burst-wrapper
+[root@hawkeye ~]# io_burst-wrapper/io_burst/burst_io.sh
+
+The script will set the sizings based on the hardware it is being run.
+
```

Comment:
````suggestion
```
````

Reply:
Trying to limit the number of ``` in the file.  The older folks are going to READ the README via cat or more

Reply:
Why?  It doesn't add visual clutter, and it makes looking at the README on GitHub nicer (someone would look at the README on GitHub at least when looking at the repo for the first time)